### PR TITLE
🐛 fix dupe key for banner notification

### PR DESCRIPTION
**Pull request by alex-wilmer**
fixes this:

image

### CODE REVIEW

File: src/packages/@ncigdc/dux/bannerNotification.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ let initialState = [
   {
     components: ['PORTAL'],
     level: 'INFO',
-    id: 0,
+    id: 'initial_banner',
     dismissible: true,
     message: <DismissibleBanner />,
```
Comment (cy) [inline src/packages/@ncigdc/dux/bannerNotification.js]: might need `dismissed: false` so the `<DismissibleBanner />` still shows? not sure

Comment (alex-wilmer) [inline src/packages/@ncigdc/dux/bannerNotification.js]: I don't think we do.. but I like more verbose initial state if that's an important thing

Comment (shanewilson) [inline src/packages/@ncigdc/dux/bannerNotification.js]: what is this initial state for?

**Comment by shanewilson** [comment]
should this be into master? the jira ticket says critical

**Comment by alex-wilmer** [comment]
sure

File: src/packages/@ncigdc/dux/bannerNotification.js (hunk 1 above)
Comment (cy) [inline src/packages/@ncigdc/dux/bannerNotification.js]: it's for setting that `Can't find your data?` dismissible notification that's hardcoded and not coming from the endpoint

**Comment by alex-wilmer** [comment]
Yeah it was just a key error. the nightly banner also had `id : 0`

**Pull request merged**